templates/ding-holdings.tpl.php

```diff
@@ -13,4 +13,5 @@
 $reserved_text = format_plural($reserved_count, 'There is 1 user in queue to loan the material.', 'There are @count users in queue to loan the material.');
 ?>
 <p><?php print "$total_text $reserved_text"; ?></p>
+<p><?php print $closest_loan; ?></p>
```

Comment:
I think we should ensure that `$closest_loan` is not empty before generating more DOM elements. The extra `<p>` tags may be empty if the provider does not return `available_from` information.